### ISSUE TITLE
VIA AC97 is broken in 1.0beta4

### ISSUE THREAD

**Issue by zmechanic** [crazii/SBEMU#112]
Release 1.0beta4 is broken for VIA AC97. Works fine with previous user build.

Although on the screen it shows that everything is successfully enabled, there's no sound.

Also, sequential runs of SBEMU often fails with page fault, and only rarely succeed saying that its enabled and showing parameters.

Also, there much less parameters displayed then in previous version.

**Comment by crazii** [comment]
Thanks for the feedback. what's the VIA chipset of yours?
Is the user build right before 1.0beta4 working?

**Comment by zmechanic** [comment]
Thanks for getting back to me. Mine audio is VT1617A. It's on this mobo if you need more hardware details:
https://www.jetwaycomputer.com/J7F2.html

One build before v1 is working OK.

**Comment by crazii** [comment]
So the [UserBuild_2024.04.28_03-45](https://github.com/crazii/SBEMU/releases/tag/UserBuild_2024.04.28_03-45) is working but 1.0beta4 is not, is that correct?
There's no related changes on the VIA part, so I need check which change (among the 3) causing the problem. 
Is it possible that you build from the source under Linux or WSL or MinGW?

**Comment by zmechanic** [comment]
That's correct. On the surface it looks like in starts correctly and shows sound adapter name. But no sound comes out.

The older build that works has green indicators while release has everything in grey.

Not sure how build from source on my end would help.
